src/features/training-program/training-program.data.tsx

```diff
@@ -157,6 +159,58 @@ export const contentMap: ContentMap = {
     ],
     image: jsImg,
   },
+
+  'js / front-end ru': {
+    title: 'Программа обучения',
+    content: [
+      <Paragraph>
+        Программа состоит из 3 этапов. На каждом последующем этапе могут быть установлены требования
+        для перехода на следующий уровень, которые будут описаны ниже. В данном запуске программы
+        будет предусмотрено самостоятельное обучение. Это означает, что у вас будет доступ к
+        предзаписанным вебинарам, рекомендуемым материалам и еженедельным онлайн сессиям вопросов и
+        ответов с нашими наставниками/координаторами для ответа на любые вопросы, которые у вас
+        могут возникнуть.
+      </Paragraph>,
+      <Paragraph>
+        У вас также будет возможность общаться с другими студентами и помогать друг другу решать
+        любые проблемы, с которыми вы можете столкнуться. Мы предоставим вам список тем, которые
+        должны быть рассмотрены на каждом этапе с рекомендуемыми сроками, но у вас будет свобода
+        выбора времени для просмотра лекций и выполнения заданий.
+      </Paragraph>,
+      <Paragraph>
+        <span>ОБРАТИТЕ ВНИМАНИЕ</span>, что сроки выполнения практических заданий не являются
+        рекомендацией и должны быть соблюдены.
+      </Paragraph>,
+      <Subtitle text="Внимание! Наставники на этом курсе будут первоначально назначены выпускникам RS School Stage #2." />,
```

Comment:
I suggest to delete this Subtitle. It is only for React and Angular courses

Reply:
I suppose it's relevant to task #219

Reply:
ok